src/app/modules/account/pages/spouse-info/add-spouse/add-spouse.component.html

```diff
@@ -139,7 +139,7 @@ <h2>Spouse's Personal Information</h2>
       <div sectionTitleInfo>
         <h2>Spouse's new last name</h2>
         <p class="horizontal-line">
-          This name will be printed on your new BC Services Card.
+          This name will be printed on your spouse's new BC Services Card.
```

Comment:
It's probably fine but should it say "the spouse's" rather than "your spouse's" the same way they asked us to change the child stuff? Or no?

Reply:
The difference I see is that the child might not be the AH's child where as the spouse is the AH's partner.